Gopkg.toml

```diff
@@ -77,3 +77,7 @@ ignored = ["github.com/kubernetes/repo-infra/kazel"]
 [[override]]
   name = "k8s.io/gengo"
   revision = "75356185a9af8f0464efa792e2e9508d5b4be83c"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Does it make sense to pin this to the same version that k/k uses?

Reply:
It would, except k/k does not vendor `kube-aggregator`. It's just used as an import. k/f also uses this strategy i.e. HEAD at the time of vendoring.

Reply:
Got it, thanks for the explanation.